### ISSUE TITLE
Allow moat on higher terrain

### ISSUE THREAD

**Issue by ByBurton** [Sh0wdown/UnofficialCrusaderPatch#369]
Should not be too hard to enable probably, maybe.

**Comment by Lolasik011** [comment]
@ByBurton Good idea.

**Comment by GRhin** [comment]
I think its realistic the way it is, and it adds a pro/con of getting the high ground (Pro: hieght bonus of enemy being less accurate and weaker, Con: cant dig moat)

**Comment by Rakso69** [comment]
Maybe humans shouldn't, but AI could?

**Comment by ByBurton** [comment]
It will be an activatable feature

**Comment by Russianstory** [comment]
In this case, for opponents of the moat on the higher plains, you can compromise. The moat can still be dug on the higher plains BUT only if there are five tanks with water. Well, as if for real - do not dig a ditch if you don’t pour water, do not pour water if there is no vat of water.
--
В таком случае, для противников рва на более высоких равнинах можно пойти на компромисс. Ров всё-таки можно будет рыть на более высоких равнинах НО только при наличии пяти чанов с водой. Ну как будто по-настоящему - ров не выкопать если воду не налить, воду не налить если нет чана с водой.

**Comment by Lolasik011** [comment]
> ## In this case, for opponents of the moat on the higher plains, you can compromise. The moat can still be dug on the higher plains BUT only if there are five tanks with water. Well, as if for real - do not dig a ditch if you don’t pour water, do not pour water if there is no vat of water.
> В таком случае, для противников рва на более высоких равнинах можно пойти на компромисс. Ров всё-таки можно будет рыть на более высоких равнинах НО только при наличии пяти чанов с водой. Ну как будто по-настоящему - ров не выкопать если воду не налить, воду не налить если нет чана с водой.

A lot of people will be needed for 5, water pot ...
Много людей понадобится на 5, горшок с водой ...

**Comment by Russianstory** [comment]
EXACTLY! you're right. Yes, 15 people is a good protection of the Lord .... people do not need to use it that way. THEN even more logical. The water in the castle and the water in the moat can only be if there is a natural source of water, that is, a well, and not a vat. So even better and more logical. In this case, it will be necessary to 5 wells for the pit. And that means only 5 people. Well, this is an example.
--
ТОЧНО! ты прав. Да 15 человек это неплохая защита лорда....людей не надо так использовать. Тогда ещё логичнее. Вода в замке и вода во рву может быть только если есть естественный источник воды то есть колодец, а не чан. Так даже лучше и логичнее. в таком случае надо будет 5 колодцев для рва. А это значит всего 5 человек. Ну это как пример.

**Comment by Russianstory** [comment]
It would not be superfluous if the list would be a list / table of what has already been done in Mod. And it is a separate list of what can be fully realized!

Is it possible to make strips of lives for units a little different?
What I want:
1) from 100% of life to 75% of the color of the strip is saturated with brightly juicy green
2) from 75% of life to 50% of the color of the strip is yellow
3) from 50% of life to 25% of the color of the strip is saturated orange
4) from 25% of life to 0% of color is juicy saturated red.

I think it would be easy for me to immediately notice those who need to be removed from the battlefield for treatment and know who to attack first - if these are enemies. Please consider my idea.
-
Было бы не лишним если был бы список именно список/таблица того, что уже сделано в Моде. и Отдельный именно список того, что вполне можно реализовать!

Можно ли сделать полоски жизней у юнитов немног другими?
ЧТо я хочу:
1) от 100% жизни до 75 % цвет полоски насыщенный ярко сочно зелёный
2) от 75% жизни до 50 % цвет полоски жёлтый
3) от 50% жизни до 25 % цвет полоски насыщенный оранжевый
4) от 25% жизни до 0% цвет сочный насыщенный красный. 

Думаю мне было бы легко сразу замечать тех, кого надо убирать с поля боя для лечения и знать кого атаковать первым - если это враги. Пожалуйста рассмотрите мою идею.

**Comment by Gaaammmler** [comment]
@Russianstory  i think you can change that in the GM1 Files, there is the Color for the Heatlh Bars
![show](https://user-images.githubusercontent.com/5760157/57653716-c1695b00-75d2-11e9-9061-bd63e7cabe13.JPG)

**Comment by GRhin** [comment]
@Russianstory good ideas, but maybe move to new issues so this can be kept about moat on higher terrain.
I like the idea of wells etc supplying the moat, however i wonder if it unnecessarily complicates the game.
Maybe as a compromise you could build a ditch which would block cavalry and seige engines, but allow footsoldiers through? This feels a little more realistic too. (maybe put some wooden spikes in the graphics for it.

**Comment by brotolisk** [comment]
> 
> 
> Maybe humans shouldn't, but AI could?

I agree with this as some AI castles dont work if there is no moat.
humans being able to make moats on high ground should be optional though.

**Comment by Russianstory** [comment]
@Gaaammmler Can you please throw me these unpacked files the strips of unit life. All stages of the strip. I will redo her.

**Comment by Gaaammmler** [comment]
@Russianstory, please write me with an issue on my Converter Side https://github.com/Gaaammmler/Gm1KonverterCrossPlatform/issues , you talk about other things than the created issue here, i will help you with my programm, but please write me an Issue for my Programm so i can help you.

**Comment by Monsterfisch** [comment]
the idea of moat on higher terrain has been raised before and i personally believe having a drymoat on higher terrain would make a lot of sense i also made an example tile in a previous thread so if its possible to add new tiles etc to the game to support this idea i would make the tiles for it no problem
